apiml-common/src/main/java/com/ca/mfaas/product/web/HttpConfig.java

```diff
@@ -92,8 +98,11 @@ public void init() {
 
             factory.setSystemSslProperties();
         }
+        catch (HttpsConfigError e) {
+            System.exit(1); // NOSONAR
+        }
```

Comment:
If you skip this exception, so, please, make sure that before every throwing of it was created a message in the log about this exception. Because I found some places without logging message before exception was throw.

Reply:
Yes, not all HttpsConfigError exceptions generate log messages. I can create a few more messages that are missing.

Reply:
Added messages to missing exceptions